CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 #### Added
 
+- `show-config` subcommand to display currently used configuration
```

Comment:
This should go under `Cast` subsection, not `Forge` :)

Reply:
oops didn't see it